miden-tx/build.rs

```diff
@@ -18,13 +17,6 @@ const MASM_FILE_EXTENSION: &str = "masm";
 fn main() -> Result<()> {
     // re-build when the MASM code changes
     println!("cargo:rerun-if-changed={ASM_DIR}");
-    println!("cargo:rerun-if-changed={KERNEL_ERRORS_FILE}");
-    println!("cargo::rerun-if-env-changed=BUILD_KERNEL_ERRORS");
-
-    // Skip this build script in BUILD_KERNEL_ERRORS environment variable is not set to `1`.
-    if env::var("BUILD_KERNEL_ERRORS").unwrap_or("0".to_string()) == "0" {
-        return Ok(());
-    }
```

Comment:
Can you also update the doc comment of the main function here to remove the reference to this env var?

Reply:
Marking this as resolved since the approach changed.